include/picongpu/plugins/adios/ADIOSWriter.hpp

```
@@ -679,6 +682,9 @@ namespace picongpu
                     if(!traits::IsFieldDomainBound<T>::value)
                     {
                         DataConnector& dc = Environment<>::get().DataConnector();
+                        // Skip PML fields when PML is not enabled
+                        if(!dc.hasId(T::getName()))
```

same as above sounds to me that we silently ignore all unknown fields.